duckdb-vortex/src/include/vortex_common.hpp

```diff
@@ -104,25 +104,25 @@ struct VortexArray {
 	vx_array *array;
 };
 
-struct VortexArrayStream {
-	explicit VortexArrayStream(vx_array_stream *array_stream) : array_stream(array_stream) {
+struct VortexArrayIter {
```

Comment:
Full name `ArrayIterator`, don't shorten to `Iter` since we don't anywhere in the Rust APIs

Reply:
Also why the Vortex prefix? C++ has namespaces!!